fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/mixin/client/keybinding/MixinKeyBindingAccessor.java

```diff
@@ -0,0 +1,13 @@
+package net.fabricmc.fabric.mixin.client.keybinding;
+
+import net.minecraft.client.options.KeyBinding;
+import net.minecraft.client.util.InputUtil;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(KeyBinding.class)
+public interface MixinKeyBindingAccessor
```

Comment:
`KeyCodeAccessor` would be a better name for this accessor interface

Reply:
Fixed.